.github/workflows/full-check.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java-version: [ '8', '11', '17' ]
+        java-version: [ '11', '17' ]
```

Comment:
Can you re-add 21 in your branch? with the asm update, this should work correctly.